pandas/tests/resample/test_datetime_index.py

```diff
@@ -122,7 +122,9 @@ def test_resample_integerarray():
 
     result = ts.resample("3T").mean()
     expected = Series(
-        [1, 4, 7], index=pd.date_range("1/1/2000", periods=3, freq="3T"), dtype="Int64"
+        [1, 4, 7],
+        index=pd.date_range("1/1/2000", periods=3, freq="3T"),
+        dtype="float64",
```

Comment:
Changed the expected dtype. I think the old test was incorrect, as `mean` should be float.

Reply:
hmm, no it tries to cast back originally (which i suppose is dubious), but yeah we should *also* return float on nullable mean ops (thought we did genreally)

Reply:
> I think the old test was incorrect, as mean should be float.

Yes, I agree that should be float.